devlib.sh

```diff
@@ -9,4 +9,9 @@ alias bench='./bin/benchmark -c=64 -r=1024 -k=scripts/insolard/configs/root_memb
 check_same_pulse() { # network launched with same pulse
     find . -name "*.log" -exec grep -m1 --mmap persist {} \; | \
     perl -ne '/current_pulse=(\d+)/; print "$1 ";'
+}
+
+get_pprof_web() {
+    curl http://localhost:8005/debug/pprof/profile\?seconds\=20 >tmp.prof
```

Comment:
what the point in helper with hardcoded url and profile time?

Reply:
right now this is a just template